### DISCUSSION TITLE
Proxy question

### DISCUSSION THREAD

**Discussion by caywin25** [searxng/searxng#3255] [Q&A] [answered]
Hello,

I have a question about the proxy please.

I know there is image_proxy mode for images.
Which proxy is used please?
But is there a proxy for the rest of searxng?
For videos? Research etc...?

Thanks

**Comment by return42** [comment, marked as answer]
In research tasks and displaying images in the result lists SearXNG itself acts as the proxy .. streams linked in the result lists are not proxied .. some maintainers configure the hostname_replace to replace e.g. youtube links to alternatives like [invidious](https://github.com/searxng/searxng/discussions/911) or piped.

**Reply by caywin25** [reply]
Hello @return42 

ok for hostname_replace but if searxng is hosted in my house, searches carried out on google (for example)  in searxng will see the IP of my house?

**Reply by return42** [reply]
yes.

**Comment by caywin25** [comment]
Ok thank you :)
